docs/getting-started/contributing/contributing-with-git.md

```diff
@@ -0,0 +1,196 @@
+
+Community contributions are an important part of USGS open source projects.  Contributions from people like you help make our projects at the USGS better for everyone.  And, by contributing, you can build your skills, display your work, and form community connections.
+
+This guide walks through the process of using git.  Git is the system the USGS uses to manage changes and contributions to our software projects.
+
+If you aren't comfortable contributing code, you can also help by contributing documentation or tutorials, or [opening](../../how-to-guides/software-management/guidelines-for-reporting-issues.md) and commenting on git issues.
+
+???+ abstract "Git Contribution Process Overview"
+    ```mermaid
+    flowchart TD
+        A(fa:fa-cloud upstream/main*\nProject Repo) -->|Other Changes to upstream/main\nFETCH & MERGE with your branch| E
+        A -->|FORK| B
+        B(fa:fa-cloud origin/main\nYour Remote Repo) -->|CLONE| C
+        C[fa:fa-computer main\nLocal Repo] -->|BRANCH| D
+        D[fa:fa-computer feature-branch\nLocal Repo] -->|Edit, Save, Test & Make COMMITS| E
+        E[fa:fa-computer feature-branch\nLocal Repo with changes] -->|PUSH| F
+        F(fa:fa-cloud origin/feature-branch\nYour Remote Repo with changes) -->|Open PULL REQUEST, Pull Request MERGED| G
+        G(fa:fa-cloud upstream/main \nProject Repo with your changes)
+    ```
+    * Note, for some projects, the working branch may be `master` or `dev` instead of main.
+
+## Prerequisites
+
+### Installing Git
+
+***git*** is a version control system for managing a project's codebase or documentation, its history, and various versions and contributions.
+
+If you don't have git yet, install as appropriate for your system.
+
+```sh
+# Run to see if/what version of git is installed
+git version
+```
+
+
+??? tip "Installing git for Windows, Mac, and Linux" 
+
+    | System  | Installation                                                                                                                                                                                               |
+    |---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Windows | Download and install git for Windows from the [git-scm website (external)](https://git-scm.com/download/win).  If you have winget you may install git via `winget install --id Git.Git -e --source winget` |
+    | Mac     | Mac installs git automatically when you run it from the terminal.  If you have Homebrew, you may install git via `brew install git`.                                                                       |
+    | Linux   | Install git using your package manager. For example, `sudo apt install git` on Ubuntu.                                                                                                                     |
+
```

Comment:
The text above directs users to install git (which is terrific). This table then tells them how to do it in a very truncated form. Is is desirable to have users posing questions about installation to the maintainers of these docs? I doubt it and would suggest removing this table.